### DISCUSSION TITLE
[BUG] cannot update to nightly build: Update-Module: Module 'PnP.PowerShell' was not installed by using Install-Module, so it cannot be updated.

### DISCUSSION THREAD

**Discussion by khha** [pnp/powershell#3587] [General] [answered]
I'm currently on PnP Powershell 2.2.0
I'm trying to update it to nightly build
Update-Module PnP.PowerShell -Scope CurrentUser -AllowPrerelease -Force

I get the following error, not sure how to go about upgrading -
Update-Module: Module 'PnP.PowerShell' was not installed by using Install-Module, so it cannot be updated.

![pnp update to nightly build](https://github.com/pnp/powershell/assets/9421415/465d2b1e-5e8e-43ce-8670-8ac576366d28)

Any ideas ?

**Comment by AndersRask** [comment, marked as answer]
Try using Install-Module instead of update-module

I'm guessing that the specific scope you are installing wasn't initially installed using Install-module (eg copy to module directory) and hence can't be updated.

**Comment by khha** [comment]
> Try using Install-Module instead of update-module
> 
> I'm guessing that the specific scope you are installing wasn't initially installed using Install-module (eg copy to module directory) and hence can't be updated.

Thanks, this works! I see that there are 2 commands for nightly build. That was my mistake. First I need to install it ofcource.

Install nightly build documentation: 
https://pnp.github.io/powershell/articles/installation.html#nightly-build

Update nightly build documentation:
https://pnp.github.io/powershell/articles/installation.html#nightly-build-1
